operator/pkg/values/translation/translation.go

```diff
@@ -123,40 +124,124 @@ func GetGlobalMonitoring(m *platform.GlobalMonitoring) *ValuesBuilder {
 	return &globalMonitoring
 }
 
-// SetScannerAnalyzerValues sets values in "sv" based on "analyzer".
-func SetScannerAnalyzerValues(sv *ValuesBuilder, analyzer *platform.ScannerAnalyzerComponent) {
-	if analyzer.GetScaling() != nil {
-		scaling := analyzer.GetScaling()
-		sv.SetInt32("replicas", scaling.Replicas)
-
-		autoscaling := NewValuesBuilder()
-		if scaling.AutoScaling != nil {
-			switch *scaling.AutoScaling {
-			case platform.ScannerAutoScalingDisabled:
-				autoscaling.SetBoolValue("disable", true)
-			case platform.ScannerAutoScalingEnabled:
-				autoscaling.SetBoolValue("disable", false)
-			default:
-				autoscaling.SetError(fmt.Errorf("invalid spec.scanner.replicas.autoScaling %q", *scaling.AutoScaling))
-			}
+// SetScannerComponentDisableValue sets the value for the 'disable' key for scanner values
+func SetScannerComponentDisableValue(sv *ValuesBuilder, scannerComponent *platform.ScannerComponentPolicy) {
+	if scannerComponent != nil {
```

Comment:
nit: invert the `if` and then `return` to reduce indentation

Reply:
Would agree. In my opinion this would also make the code style more consistent.

Reply:
Done.